### PR TITLE
Release v1.3.0

### PR DESCRIPTION
**Pull request by pinin4fjords**
Pre-release operations:

- Version bumping
- Citation updates
- Modules updates

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/differentialabundance/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/differentialabundance _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -384,7 +384,7 @@ manifest {
     description     = 'Differential abundance analysis'
     mainScript      = 'main.nf'
     nextflowVersion = '!>=23.04.0'
-    version         = '1.3.0dev'
+    version = '1.3.0'
```
Comment (jonasscheid) [inline nextflow.config:387]: Small nitpick: Algin spacing here

**Review by jonasscheid — APPROVED** [review]
Minor: Adjust spacing of version in nextflow.config

else, LGTM 👍🏼 !

**Review by Emiller88 — APPROVED** [review]
Looks good besides @jonasscheid's nit

File: nextflow.config (hunk 1 above)
Comment (pinin4fjords) [reply nextflow.config:387]: Dammit- this is `nf-core bump-version`'s doing!

**Comment by pinin4fjords** [comment]
Thanks @jonasscheid @Emiller88 !

**Pull request merged**